apps/src/tutorialExplorer/filterGroupContainer.js

```diff
@@ -0,0 +1,44 @@
+/* FilterGroupContainer: The container UI for a group of filter choices.
```

Comment:
Components are usually .jsx files. Does it matter that this one is .js?

Reply:
.js is fine, but ideally it should be FilterGroupContainer (first letter capitalized)

Reply:
Good catch.  tutorialExplorer doesn't use first letter capped at the moment but could change in a broader change later...